java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -90,57 +82,4 @@ public void info_with_everything_option() {
             assertTrue(data.contains("# " + section), "Section " + section + " is missing");
         }
     }
-
-    @Test
-    @SneakyThrows
-    public void set_and_get_without_options() {
```

Comment:
why?

Reply:
Why what? Sneaky throws or why did I remove them? If it is the latter these are tests meant to be in baseClient tests not only the cluster I transfered them to there.